CHANGELOG.md

```diff
@@ -551,6 +581,23 @@ autopilot to fail to discover new server versions and so not trigger an upgrade.
 * ui: fixed a bug where the replication pages did not update display when navigating between DR and performance [[GH-26325](https://github.com/hashicorp/vault/pull/26325)]
 * ui: fixes undefined start time in filename for downloaded client count attribution csv [[GH-26485](https://github.com/hashicorp/vault/pull/26485)]
 
+## 1.16.11
```

Comment:
This is an enterprise-only release, right? If so (at least according to our instructions) I think this needs to have Enterprise after it. Same with the 1.15.16 one.

Reply:
AH that is where it is needed ty. is the 1.17.7 on an ent only as well?

Reply:
Oh I think you're right yes, because we only release N for CE now, not even N-1.

Reply:
done

Reply:
Looking here yes, only the enterprise binary is available: https://releases.hashicorp.com/vault/